analyses/fusion_filtering/02-fusion-filtering.R

```diff
@@ -179,8 +175,16 @@ saveRDS(QCFiltered,paste0(opt$outputfile,"_QC_filtered.RDS"))
 
 # load expressionMatrix RDS for expression based filtering for less than given threshold
 expressionMatrix<-readRDS(expressionMatrix)
-expressionMatrix <- cbind(expressionMatrix, colsplit(expressionMatrix$gene_id, pattern = '_', names = c("EnsembleID","GeneSymbol")))
-
+# find the list of cohorts and sample type of interest
+matched_samples <- read.delim(clinicalFile, header = TRUE, sep = "\t", stringsAsFactors = FALSE) %>%
+  filter(cohort == "PBTA" | cohort == "GMKF") %>%
```

Comment:
maybe read this as input arg, incase we need to update this selection it can be done directly on the bash script.

Reply:
I added clinicalFile<-opt$clinicalFile - is that what you mean by reading this as input arg? or do you mean by making the cohort an input arg?

Reply:
make the `cohort` as a input arg. Since each cohort would need to use a different normal_expression_file, like PBTA would use a normal expression file which filters with `gtex_group=="Brain"` and GMKF will use `gtex_group=="Adrenal Gland"`